### PR TITLE
Zoisite–Yael Peet & Zhuo F–Solar-system–wave 1 & 2

### PR DESCRIPTION
**Pull request by Strawzz**

### CODE REVIEW

**Comment by yangashley** [comment]
@Strawzz @yaelsilvermanpeet Great work on waves 1 and 2 for Solar System. It looks like you've got a handle on creating a blueprint, registering it with your Flask app and writing routes. Nice job refactoring your code so that the routes can stay short and sweet!

File: app/routes.py [hunk 1]
```diff
@@ -1,2 +1,58 @@
...
+
+    planet = validate_planet_id(planet_id)
+
+    return planet.make_dict()
+
+
+@planet_bp.route("", methods=["GET"])
+def handle_planets():
+
+    results = [planet.make_dict() for planet in planets]
```
[the remark this answers is not in the document]
Comment (yaelsilvermanpeet) [reply app/routes.py]: you literally suggested it ;)

File: tests/test_models.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    # Arrange
+    planet_data = {
+        "name": "Endor"
+    }
+
+    # Act & Assert
+    with pytest.raises(KeyError, match = 'description'):
+        new_planet = Planet.from_dict(planet_data)
+
+def test_from_dict_with_extra_keys():
```
Comment (kelsey-steven-ada) [inline tests/test_models.py:101]: Nice tests for the model!

File: tests/test_routes.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    assert response_body[0] == {
+        "id": 1,
+        "name" : "Tatooine",
+        "description" : "Teenage"
+    }
+    assert response_body[1] == {
+        "id": 2,
+        "name": "Hoth",
+        "description": "You're a cold as ice"
+    }
```
Comment (kelsey-steven-ada) [inline tests/test_routes.py:28]: Great choice to check the whole response body to ensure all values are what you expect!

**Review by kelsey-steven-ada — COMMENTED** [review]
Great work Zhuo & Yael! I've left a few comments & questions, feel free to reply here or message me on Slack if you have questions on the feedback 🙂

**Pull request closed without merging**